### PR TITLE
PR to add anisble-python-jobs to ansible.security

### PR DESCRIPTION
**Pull request by justjais**
PR to add anisble-python-jobs to ansible.security

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -66,6 +66,8 @@
     merge-mode: squash-merge
     templates:
       - system-required
+      - ansible-python-jobs
+      - publish-to-automation-hub
```
Comment (pabelanger) [inline zuul.d/projects.yaml:70]: we should add galaxy here too

Comment (justjais) [reply zuul.d/projects.yaml:70]: @pabelanger added in PR #843

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pabelanger) [inline .idea/workspace.xml:1]: needs to be deleted

Comment (justjais) [reply .idea/workspace.xml]: ah.. deleted

**Pull request merged**